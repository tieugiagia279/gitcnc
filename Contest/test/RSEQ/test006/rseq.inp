1000 100
412 79 686 668 876 366 195 279 718 430 784 815 90 447 863 954 270 315 965 792 588 325 859 72 737 276 913 341 216 944 774 407 244 986 635 798 157 653 864 497 16 722 909 350 317 720 251 765 9 272 487 375 626 553 616 271 153 432 499 285 619 550 571 425 103 504 551 424 730 797 122 818 446 404 197 572 405 91 929 820 641 108 864 178 467 75 292 802 367 117 90 889 877 92 271 980 106 386 358 408 377 321 325 761 909 91 823 11 206 933 469 711 422 916 694 811 97 229 145 459 410 640 980 380 915 161 990 478 201 931 39 46 625 608 899 295 385 245 816 624 592 819 917 281 879 50 325 866 30 569 755 575 138 114 742 73 609 316 552 181 547 680 180 702 525 439 39 929 365 4 16 397 675 880 338 51 282 585 599 588 859 153 560 589 966 124 287 859 11 863 7 21 581 806 268 801 537 841 161 360 431 828 480 956 359 58 990 781 533 682 684 617 39 123 631 732 907 878 797 219 795 325 767 436 991 184 640 677 122 640 986 955 505 147 552 788 882 790 937 630 757 971 422 822 440 925 39 586 462 636 472 483 476 476 916 571 464 244 716 269 157 938 134 529 645 556 44 618 742 62 472 105 201 729 489 294 347 187 924 682 222 171 685 124 513 557 803 101 62 628 285 10 133 917 730 786 824 818 925 91 946 124 628 631 453 306 836 680 124 130 558 530 820 445 447 796 13 889 554 333 602 250 177 896 800 916 141 650 259 212 66 613 720 3 379 5 142 416 159 698 749 48 775 223 576 508 420 142 732 866 436 952 118 390 791 476 103 922 789 121 267 320 478 913 665 114 36 708 16 642 286 344 585 545 903 833 225 909 146 472 339 253 192 498 939 952 566 892 744 35 725 770 269 76 811 678 17 674 526 943 37 402 907 839 759 28 963 152 304 519 122 707 957 763 61 166 473 57 100 162 792 153 644 886 37 185 86 234 148 892 914 902 130 411 491 939 528 8 376 413 319 548 551 826 475 525 592 450 949 352 28 88 233 529 863 113 200 320 149 144 332 985 726 608 144 945 235 952 93 400 970 207 186 108 313 76 311 14 457 251 238 263 731 509 993 545 380 195 306 550 176 488 325 923 762 944 612 371 848 881 307 262 724 559 334 929 775 750 99 439 802 309 288 448 886 774 831 411 773 150 891 335 995 51 552 945 181 756 122 489 807 362 928 869 985 7 308 917 208 262 931 980 267 986 113 615 51 881 483 771 383 193 534 308 779 269 69 44 224 977 672 611 654 953 318 455 771 459 351 246 242 577 556 753 998 215 622 610 957 53 727 161 270 481 62 521 581 370 415 992 25 346 14 530 560 839 275 318 813 322 673 6 125 299 369 342 887 357 905 193 959 395 661 718 553 245 188 529 744 172 715 234 332 854 502 683 279 525 532 895 439 503 283 821 569 658 667 917 824 154 954 628 976 590 244 807 152 139 634 876 943 594 33 867 563 537 579 832 24 38 499 769 510 816 494 419 664 97 124 992 325 519 947 567 740 163 966 907 658 515 817 913 111 899 444 136 427 424 750 968 369 959 957 355 335 810 991 411 198 225 540 923 477 102 44 658 860 246 155 41 511 334 713 677 609 50 390 532 796 106 837 474 743 30 595 466 499 450 378 836 653 353 10 485 847 323 986 753 35 752 815 118 571 445 853 729 875 449 359 24 303 928 109 687 620 206 546 624 518 523 597 427 157 297 147 31 715 516 959 956 694 379 13 131 318 604 431 592 846 217 813 431 966 492 0 545 58 298 517 198 706 354 887 364 542 17 457 947 172 467 845 244 481 532 252 474 165 623 593 465 588 822 946 980 325 691 841 525 456 80 506 94 693 27 313 926 533 639 456 915 618 229 218 982 334 929 308 538 784 300 36 83 430 673 83 298 292 701 438 869 78 777 241 93 812 462 546 114 577 56 539 196 553 398 444 112 820 434 69 17 996 653 665 691 176 275 763 762 193 918 836 30 109 523 459 455 458 681 71 518 540 816 339 244 181 11 587 838 235 118 403 854 888 320 430 331 211 972 468 196 143 811 939 26 657 671 273 368 315 580 400 303 907 695 888 821 543 854 553 610 418 744 626 399 348 126 220 557 926 310 219 359 355 249 873 535 374 692 732 190 934 388 576 12 458 955 142 712 265 725 903 779 817 637 200 913 935 676 319 73 235 6 780 56 139 327 367 850 75 93 628 223 493 796 25 872 675 170 323 806 616 897 851 730 143 801 153 311 114 365 
292 428 931 378 124 602 468 916 685 31 671 169 23 248 247 941 596 261 132 802 246 846 911 537 465 155 158 950 130 515 361 694 189 737 211 993 526 662 278 631 55 541 990 443 765 675 705 490 580 320 710 236 234 624 586 299 4 811 220 826 660 463 421 345 658 655 319 167 111 473 386 532 565 174 572 140 642 41 360 687 757 907 822 568 88 37 396 961 73 683 327 922 953 524 69 550 601 620 441 625 