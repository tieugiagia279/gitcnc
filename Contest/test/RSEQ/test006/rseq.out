350049
284540
250949
250949
250949
165138
165138
156357
110445
110445
110445
110445
110445
110445
110445
110445
110445
110445
110445
63232
63232
63232
63232
56989
56989
56989
56989
56989
56989
56989
56989
51644
51644
44055
44055
44055
44055
44055
44055
44055
33708
33708
33708
33708
33708
33708
33708
33708
33708
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
31826
28852
28852
28852
28852
28852
28852
28852
28852
28852
28852
28852
28852
28852
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
27005
